src/server_manager/test.webpack.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 The Outline Authors
```

Comment:
nit: maybe rename the file `to web_app_test.webpack.js`

Reply:
Thanks, that's a much better name. I ended up just removing this file completely since it didn't provide much value.